doc/man3/DES_random_key.pod

```diff
@@ -282,6 +282,24 @@ functions directly.
 Single-key DES is insecure due to its short key size.  ECB mode is
 not suitable for most applications; see L<des_modes(7)>.
 
+=head1 RETURN VALUES
+
+DES_set_key(), DES_key_sched(), DES_set_key_checked() and DES_is_weak_key()
+return 0 on success or negative values on error.
+
+DES_random_key(), DES_set_key_unchecked(), DES_set_odd_parity(), DES_ecb_encrypt(),
+DES_ecb2_encrypt(), DES_ecb3_encrypt(), DES_ncbc_encrypt(), DES_cfb_encrypt(),
+DES_ofb_encrypt(), DES_pcbc_encrypt(), DES_cfb64_encrypt(), DES_ofb64_encrypt(),
+DES_xcbc_encrypt(), DES_ede2_cbc_encrypt(), DES_ede2_cfb64_encrypt(),
+DES_ede2_ofb64_encrypt(), DES_ede3_cbc_encrypt(), DES_ede3_cfb64_encrypt(),
+DES_ede3_ofb64_encrypt(), DES_string_to_key() and DES_string_to_2keys() do not
+return values.
+
+DES_cbc_cksum() and DES_quad_cksum() return 4-byte integer.
+
+DES_fcrypt() and DES_crypt() return a null-terminated string on success or NULL
+if an error occurred.
```

Comment:
Hmm.... I'm struggling with "a null-terminated string" here. I mean "a" kind of implies "some", while it's quite specific. In DES_fcrypt case it's either buffer supplied by caller or NULL, and in DES_crypt - static buffer or NULL.

[On *side note* one can wonder if it would be appropriate to not modify buffer if error is returned. I mean DES_fcrypt modifies buffer in case error is returned, and maybe it shouldn't. Once again, this is *side note*.]

Reply:
Or could we say 'null terminated buffer' or something like that?

Reply:
Just in case of misunderstanding, I'm struggling specifically with "a", i.e. with it being *indefinite* article. And underlying question rather is if one should be more specific than referring to unnamed buffer, with or without article. After all, description says nothing about return value, only input arguments, so it might be appropriate to say explicitly how it is. So that reader doesn't have to consult source code...

Reply:
How about this:

```
DES_fcrypt() and DES_crypt() return NULL if an error occurred. On suucess, DES_fcrypt() returns a null-terminated string in user provided buffer and DES_crypt() returns a null-terminated string in a static buffer.
```

Reply:
Well, to me it appears more of a function description than description of return value. I mean part of placing null-terminated string to a buffer. Consider 'man strcpy' for example... So I'd say it should rather read "On success DES_fcrypt() returns pointer to the caller-provided buffer, and DES_crypt() - to a static buffer."

Or related *side* note one can wonder if DESCRIPTION part needs some adjustments. Most notably mention that returned string is null-terminated. [On related note one can also wonder if "this version takes only a small amount of space" "bragging" clause is actually relevant today.]

Reply:
This is fixed and the 'null-terminated' description is added to section DESCRIPTION.